tests/test_pipelines.py

```diff
@@ -55,14 +62,14 @@
 TEXT_CLASSIF_FINETUNED_MODELS = {
     (
         "distilbert-base-cased",
-        "distilbert-base-uncased-finetuned-sst-2-english",
-        "distilbert-base-uncased-finetuned-sst-2-english",
+        "sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english",
+        "sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english",
     )
 }
 
 TEXT_GENERATION_FINETUNED_MODELS = {
-    ("gpt2", "gpt2"),
-    ("xlnet-base-cased", "xlnet-base-cased"),
+    # model = tokenizer
+    "sshleifer/tiny-gpt2",
```

Comment:
Can you take advantage of this PR to also move the other ones to just a model identifier?

Reply:
Will make things cleaner